model-engine/model_engine_server/api/app.py

```diff
@@ -23,9 +23,8 @@
 from model_engine_server.api.triggers_v1 import trigger_router_v1
 from model_engine_server.core.loggers import (
     filename_wo_ext,
-    get_request_id,
     make_logger,
-    set_request_id,
+    LoggerTagManager, LoggerTagKey
```

Comment:
Does this pass the linter?

Reply:
Probably not, my pre-commit setup is a little wonky with submodules right now.